sql-statements/sql-statement-create-table.md

```diff
@@ -83,7 +83,7 @@ TableOption ::=
 |   'SECONDARY_ENGINE' EqOpt ( 'NULL' | StringName )
 |   'UNION' EqOpt '(' TableNameListOpt ')'
 |   'ENCRYPTION' EqOpt EncryptionOpt
-|    'TTL' EqOpt TimeColumnName '+' 'INTERVAL' Expression TimeUnit (TTLEnable EqOpt ( 'ON' | 'OFF' ))?
+|    'TTL' EqOpt TimeColumnName '+' 'INTERVAL' Expression TimeUnit (TTLEnable EqOpt ( 'ON' | 'OFF' ))? (TTLJobInterval EqOpt stringLit)?
```

Comment:
This is [the preview of the diagram](https://test-ebnf-diagram.pingcap-docs-website-scaffold-1hq.pages.dev/zh/tidb/dev/sql-statement-create-table). Please make sure that the rendering is correct. Thanks!

Reply:
Thanks. It's correct :+1: .